python/py-pylint/Portfile

```
@@ -6,9 +6,9 @@ PortGroup           select 1.0
 
 name                py-pylint
 version             2.17.4
-revision            0
+revision            1
```

No need to increase the revision since there's no change in the installed files or dependencies.
```suggestion
revision            0
```